lib/Elevate/Blockers/Repositories.pm

```diff
@@ -63,24 +64,19 @@ sub _blocker_invalid_yum_repos ($self) {
 
         for my $unsupported_repo ( @{ $self->{_yum_repos_unsupported_with_packages} } ) {
             my $blocker_id = ref($self) . '::' . $unsupported_repo->{'name'};
-            $self->has_blocker( $unsupported_repo->{'json_report'}, 'blocker_id' => $blocker_id, 'quiet' => 1 );
+            $self->has_blocker(
+                $msg,
```

Comment:
It appears that if neither `INVALID_SYNTAX` or `USE_RPMS_FROM_UNVETTED_REPO` are set, then the `$msg` will be empty.  There probably ought to be a default message.

Reply:
This code is unreachable unless either `INVALID_SYNTAX` or `USE_RPMS_FROM_UNVETTED_REPO` are set.

` 63         return 0 unless _yum_status_hr_contains_blocker($status_hr);`

^ This will return prior to reaching this code if neither are set.

Reply:
Okay, cool.